### PR TITLE
Add new login page

### PR DESCRIPTION
**Pull request by adnansh95**
--- Why is this change necessary?
We need a new wonderful login page

--- How does it address the issue?
Edit the default login page and edit also the application.scss file

### CODE REVIEW

File: src/dokumentvwsys/app/views/devise/sessions/new.html.erb [hunk 1]
```diff
@@ -1,26 +1,19 @@
...
 
-  <div class="field">
-    <%= f.label :password %><br />
-    <%= f.password_field :password, autocomplete: "current-password" %>
-  </div>
+    <form>
+      <input type="text" id="login" class="fadeIn second" name="login" placeholder="login">
+      <input type="text" id="password" class="fadeIn third" name="login" placeholder="password">
+      <input type="submit" class="fadeIn fourth" value="Log In">
+    </form>
```
Comment (mkapra) [inline src/dokumentvwsys/app/views/devise/sessions/new.html.erb:12]: Where is the rails form?

File: src/dokumentvwsys/app/assets/stylesheets/application.scss [hunk 2]
```diff
@@ -4,3 +4,266 @@
...
+/* OTHERS */
+
+*:focus {
+    outline: none;
+} 
+
+#icon {
+  width:60%;
+}
+
```
Comment (mkapra) [inline src/dokumentvwsys/app/assets/stylesheets/application.scss:269]: This is bad. It is for all sites and not only the login site

File: .gitignore [hunk 3]
```diff
@@ -1 +1,2 @@
 .idea
+*~
```
Comment (mkapra) [inline .gitignore:2]: ?

**Review by mkapra — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 3 above)
Comment (adnansh95) [reply .gitignore:2]: to ignore all temp files

File: src/dokumentvwsys/app/views/devise/registrations/new.html.erb [hunk 4]
```diff
@@ -1,44 +1,36 @@
-<h2>Sign up</h2>
```
Comment (mkapra) [inline src/dokumentvwsys/app/views/devise/registrations/new.html.erb:1]: I think this heading should stay

File: src/dokumentvwsys/app/views/devise/registrations/new.html.erb [hunk 5]
```diff
@@ -1,44 +1,36 @@
...
-
-  <div class="actions">
-    <%= f.submit "Sign up" %>
+  <div class="container row col-sm-9 col-md-7 col-lg-5 mx-auto">
+    <div class="card card-signin my-5 card-body">
+      <form class="form-signup">
+        <div class="form-label-group">
+          <div class="center_div"><img src="/hi.png" id="icon" alt="User Icon" class: "mx-auto d-block"/></div>
+          <%= f.label :username %><br />
+          <%= f.text_field :username, autofocus: true, autocomplete: "username", class:"form-control" %>
```
Comment (mkapra) [inline src/dokumentvwsys/app/views/devise/registrations/new.html.erb:9]:
```suggestion
          <%= f.text_field :username, autofocus: true, autocomplete: "username", class: "form-control" %>
```

File: src/dokumentvwsys/app/views/devise/registrations/new.html.erb [hunk 6]
```diff
@@ -1,44 +1,36 @@
...
-    <%= f.date_select :birth %>
-  </div>
-
-  <div class="actions">
-    <%= f.submit "Sign up" %>
+  <div class="container row col-sm-9 col-md-7 col-lg-5 mx-auto">
+    <div class="card card-signin my-5 card-body">
+      <form class="form-signup">
+        <div class="form-label-group">
+          <div class="center_div"><img src="/hi.png" id="icon" alt="User Icon" class: "mx-auto d-block"/></div>
```
Comment (mkapra) [inline src/dokumentvwsys/app/views/devise/registrations/new.html.erb:7]: `class: "mx-auto d-block"`
This is a rails specific and will not work here

File: src/dokumentvwsys/app/views/devise/sessions/new.html.erb [hunk 7]
```diff
@@ -1,26 +1,28 @@
...
 <%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
-  <div class="field">
-    <%= f.label :username %><br />
-    <%= f.text_field :username, autofocus: true, autocomplete: "username" %>
-  </div>
+  <div class="container row col-sm-9 col-md-7 col-lg-5 mx-auto">
+    <div class="card card-signin my-5 card-body">
+      <form class="form-signin">
+        <div class="form-label-group">
+          <div class="center_div"><img src="/hi.png" id="icon" alt="User Icon" class: "mx-auto d-block"/></div>
```
Comment (mkapra) [inline src/dokumentvwsys/app/views/devise/sessions/new.html.erb:6]: This image is not centered on an iphone

**Review by mkapra — CHANGES_REQUESTED** [review]

**Review by mkapra — APPROVED** [review]

**Pull request merged**